roles/keystone/templates/etc/keystone/policy.json

```diff
@@ -1,5 +1,9 @@
 {
+{% if policy_for_horizon|default('False')|bool %}
+    "admin_required": "role:admin or is_admin:1 or role:cloud_admin",
```

Comment:
By doing this are we opening up more access to cloud_admin and causing errors on horizon when the cloud admin does not have access to a certain api. This might be okay but just wondering what the effects of this could be. This will not work for project admin role.